remove_win32_bit/jep.md

```diff
@@ -1 +1,75 @@
-TODO
+# JEP NNN: Removal of Windows 32-bit Support
+
+_Author_	Bruno Borges & George Adams
+_Owner_ Bruno Borges
+_Type_	Feature
+_Scope_	Implementation
+_Status_	Draft
+_Release_	23
+_Component_	hotspot / other
+_Discussion_	jdk dash dev at openjdk dot org
+_Effort_	?
+_Duration_	?
+_Reviewed by_	Magnus Ihse Bursie
+_Endorsed by_	Magnus Ihse Bursie
+_Created_	<date>
+_Updated_	<date>
+_Issue_	<issue#>
+
+Summary
+-------
+
+Following the deprecation of Windows 32-bit support in JEP 449, this proposal takes the next step by completely removing Windows 32-bit support from OpenJDK. 
+This decision acknowledges the evolving landscape of computing, where 64-bit systems have become predominant. 
+The removal of Windows 32-bit support will streamline development and focus resources on enhancing the performance and security of OpenJDK on modern architectures.
+
+Goals
+-----
+
+- To cease all testing and development efforts targeting the Windows 32-bit platform within the OpenJDK project.
+- To simplify OpenJDK's build and test infrastructure by focusing solely on Windows 64-bit support, aligning with current computing standards.
```

Comment:
The complete removal of all code specific to just 32-bit Windows from the code base should be mentioned as a goal. This is kind of the most important one. There is no testing going on afaik, so saying that we are dropping it is not really saying anything. Nor are there any development efforts targeting 32-bit windows. 

The entire problem here is that there is a lot of legacy 32-bit code laying around which makes people hesitant as how to treat it when they come across it and need to change it.